Tribler/Test/Core/Libtorrent/test_libtorrent_download_impl.py

```diff
@@ -140,3 +140,60 @@ def test_multifile_torrent(self):
         test_dict["a"] = "b"
         pstate.set("state", "engineresumedata", test_dict)
         return impl.network_create_engine_wrapper(pstate)
+
+    @deferred(timeout=20)
+    def test_save_resume(self):
+        """
+        testing call resume data alert
+        """
+        tdef = TorrentDef()
```

Comment:
I end up refactor those which not in my test. It should be in https://github.com/Tribler/tribler/pull/2265/commits/fb2d2698abbb778700f54e399700e5b58dff9187